addthis_fields/addthis_fields.info.yml

```diff
@@ -0,0 +1,11 @@
+name: AddThis Fields
+description: Provides a basic field definitions for rendering AddThis.
```

Comment:
Like the other submodule, it would be curious to understand the motivation here, possibly even in the addthis readme